### PR TITLE
Fix: do not change timestamps for already existing helm charts

### PR DESCRIPTION
**Pull request by perk-sumo**
The command `helm repo index` is changing timestamps for already existing charts.
This seems to be intended: https://github.com/helm/helm/issues/7363
Helm stable repo tests use this workaround:
https://github.com/helm/charts/blob/c6e503ccfd9911dd06ea9ac389a40a3490637516/test/repo-sync.sh#L83-L90

This PR adds the mentioned workaround to our build.

It's a first step for #987

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by sumo-drosiek — APPROVED** [review]

**Comment by perk-sumo** [comment]
It's a first step for https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/987

File: ci/build.sh [hunk 1]
```diff
@@ -213,16 +213,29 @@ function push_docker_image() {
...
   git checkout -- .
   sudo helm init --client-only
   sudo helm repo add falcosecurity https://falcosecurity.github.io/charts
   sudo helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-  sudo helm package deploy/helm/sumologic --dependency-update --version="$version" --app-version="$version"
+
+  # due to helm repo index issue: https://github.com/helm/helm/issues/7363
+  # we need to create new package in a different dir, merge the index and move the package back
+  mkdir -p "${sync_dir}"
+  sudo helm package deploy/helm/sumologic --dependency-update --version="$version" --app-version="$version" --destination "${sync_dir}"
```
Comment (pmalek-sumo) [inline ci/build.sh:229]:
```suggestion
  sudo helm package deploy/helm/sumologic --dependency-update --version="${version}" --app-version="${version}" --destination "${sync_dir}"
```

**Review by pmalek-sumo — COMMENTED** [review]
I wonder if we really need `helm` with `sudo` 🤔

File: ci/build.sh (hunk 1 above)
Comment (perk-sumo) [reply ci/build.sh]: 👍 added :)

**Comment by perk-sumo** [comment]
@pmalek-sumo good idea, let's try removing `sudo` in another PR.

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**